### PR TITLE
few minor typos corrected

### PR DESCRIPTION
**Pull request by rksin8**
Few minor typos corrected

### CODE REVIEW

File: book/05-more-rmarkdown.Rmd [hunk 1]
```diff
@@ -164,8 +164,7 @@ Firstly lets make sure we have our data properly filtered. We are going to filte
 
 ```{r echo = F, comment=""}
 text <- c("```{r}",
-          "df2 <- df %>% filter(Airline == 'Jetstar' | Airline == 'Qantas')",
-	  "                     Route != 'All Ports-All Ports')",
+          "df2 <- df %>% filter( (Airline == 'Jetstar' | Airline == 'Qantas') & Route != 'All Ports-All Ports')",
```
Comment (serine) [inline book/05-more-rmarkdown.Rmd]: Thanks for picking this up, it was a bug. I'm a little unsure if it meant to be AND (`&`) or simply comma? In this case I'm doing two separate filtering, one for airlines and one for route. I think what you have there is fine, but just a little unsure

**Comment by serine** [comment]
@rksin8 thanks for the PR this is awesome. Can you please reset you last commit and drop `book/libs` from it. And perhaps add that to `.gitignore` those files aren't meant to be part of the repo. They are transient file made during build. My fault for not including them into `.gitignore` earlier. 

you should be able to 

```git reset HEAD~1``` this will undo last commit (I'm assuming that `book/libs`) were part of the last commit. And then just commit everything else. Don't forget to grab your commit message so that you don't have to re-type it again with

```git --no-pager log --format=%B -n1```

Cheers!

File: book/05-more-rmarkdown.Rmd (hunk 1 above)
Comment (rksin8) [reply book/05-more-rmarkdown.Rmd]: Modified the filter syntax, replaced & with ",".

**Comment by rksin8** [comment]
> @rksin8 thanks for the PR this is awesome. Can you please reset you last commit and drop `book/libs` from it. And perhaps add that to `.gitignore` those files aren't meant to be part of the repo. They are transient file made during build. My fault for not including them into `.gitignore` earlier.
> 
> you should be able to
> 
> `git reset HEAD~1` this will undo last commit (I'm assuming that `book/libs`) were part of the last commit. And then just commit everything else. Don't forget to grab your commit message so that you don't have to re-type it again with
> 
> `git --no-pager log --format=%B -n1`
> 
> Cheers!

Updated.gitignore, also updated few more syntax observed during today workshop.

**Comment by serine** [comment]
@rksin8 thanks so much for contributing! I haven't pulled this PR in time and it has gone out of sync. I made a fair few improvements and changes to the content. I'll be sure to pull any future PR promptly

**Pull request closed without merging**